beacon-chain/db/kv/blob.go

```diff
@@ -54,7 +54,7 @@ func (s *Store) SaveBlobSidecar(ctx context.Context, scs []*ethpb.BlobSidecar) e
 		// If there is no element stored at blob.slot % MAX_SLOTS_TO_PERSIST_BLOBS, then we simply
 		// store the blob by key and exit early.
 		if len(replacingKey) != 0 {
-			slotBytes := replacingKey[:8]
+			slotBytes := replacingKey[8:16]
```

Comment:
is there a comment that can be added to direct readers on this

Reply:
@james-prysm it's in the godoc for this method. 

```
//  2. Compute key for blob as bytes(slot_to_rotating_buffer(blob.slot)) ++ bytes(blob.slot) ++ blob.block_root
```

Terence wants `bytes(blob.slot)` here, which is the second quadword

Reply:
@james-prysm what do you think about this? https://github.com/prysmaticlabs/prysm/pull/12855 

Would that the PR make it easier to understand?